src/components/AQIGauge/GaugeSVG.tsx

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import {arc} from 'd3-shape';
+import {scaleLinear} from 'd3-scale';
+import {DialProps} from './AQIDial';
+
+const GaugeSVG: ({currentAQI}: DialProps) => JSX.Element = ({
```

Comment:
```suggestion
const GaugeSvg: ({currentAqi}: DialProps) => JSX.Element = ({
```

Change file name too